data/emulator.js

```diff
@@ -738,7 +738,7 @@ class EmulatorJS {
         clearTimeout(this.msgTimeout);
         this.msgTimeout = setTimeout(() => {
             this.msgElem.innerText = "";
-        }, 3000)
+        }, (typeof time === "number") ? time : 3000)
```

Comment:
Could we verify time is also greater than 0

Reply:
Done